bugbug/model.py

```diff
@@ -113,11 +114,14 @@ def le(self):
     def get_feature_names(self):
         return []
 
-    def get_important_features(self, cutoff, shap_values):
+    def get_important_features(
+        self, cutoff, avg_shap_values, shap_values, class_names=False
```

Comment:
Move the avg calculation in this function, so you can only pass `shap_values` to it